src/Uno.UWP/Media/Playback/MediaPlayer.Android.cs

```diff
@@ -263,6 +283,15 @@ public void OnCompletion(AndroidMediaPlayer mp)
 		{
 			MediaEnded?.Invoke(this, null);
 			PlaybackSession.PlaybackState = MediaPlaybackState.None;
+
+			// Play next item in playlist, if any
+			if (_playlistIndex < _playlistItems.Count - 1)
+			{
+				_player.Reset();
```

Comment:
What about switching delay? Can we add a comment if any?
Playlist management could be a good reason to investigate ExoPlayer implementation

Reply:
Added as a "Future Improvement" in `MediaPlayerElement.md`.

ExoPlayer wraps everything in a ConcatenatingMediaSource and performs additional logic during around the `Prepare()` phase, out of scope for this PR.